src/pages/_document.tsx

```diff
@@ -3,7 +3,9 @@ import { Html, Head, Main, NextScript } from 'next/document'
 export default function Document() {
```

Comment:
Curious about this page, according to the docs it's rendered on the server side, but I don't really understand it lol. Is this where we are supposed to put the font for some reason?